docs/spec/api.md

```diff
@@ -1,3 +1,7 @@
+page_title: Docker Registry HTTP API V2
```

Comment:
@moxiegirl understood. This is one of the shortcomings we need to address with the current documentation build system (unfortunately it's too intrusive and break existing specs that are generated - thus cannot be modified - we will have to live with the "broken" version for now).
Let's talk about this and figure out a good solution.